prow/def.bzl

```diff
@@ -95,11 +103,22 @@ def target(cmd):
 #     "gcr.io/k8s-prow/ghproxy:20180203-deadbeef": "//ghproxy:image",
 #     "gcr.io/k8s-prow/ghproxy:latest": "//ghproxy:image",
 #     "gcr.io/k8s-prow/ghproxy:latest-fejta": "//ghproxy:image",
+#     "gcr.io/k8s-prow-edge/hook:20180203-deadbeef": "//prow/cmd/hook:image",
+#     "gcr.io/k8s-prow-edge/hook:latest": "//prow/cmd/hook:image",
+#     "gcr.io/k8s-prow-edge/hook:latest-fejta": "//prow/cmd/hook:image",
+#     "gcr.io/k8s-prow-edge/plank:20180203-deadbeef": "//prow/cmd/plank:image",
+#     "gcr.io/k8s-prow-edge/plank:latest": "//prow/cmd/plank:image",
+#     "gcr.io/k8s-prow-edge/plank:latest-fejta": "//prow/cmd/plank:image",
+#     "gcr.io/k8s-prow-edge/ghproxy:20180203-deadbeef": "//ghproxy:image",
+#     "gcr.io/k8s-prow-edge/ghproxy:latest": "//ghproxy:image",
+#     "gcr.io/k8s-prow-edge/ghproxy:latest-fejta": "//ghproxy:image",
 #   }
 def tags(cmds, targets):
     # Create :YYYYmmdd-commitish :latest :latest-USER tags
     cmd_targets = {prefix(cmd): target(cmd) for cmd in cmds}
+    cmd_targets.update({edge_prefix(cmd): target(cmd) for cmd in cmds})
```

Comment:
it would be useful to only push the edge targets if edge is defined.

Reply:
Updated.